deploy/templates/service-ca/crd_webhooks_patch.yaml

```diff
@@ -34,9 +33,8 @@ spec:
       conversionReviewVersions: ["v1"]
       clientConfig:
         service:
-          namespace: devworkspace-operator
-          # Note: service name is hard-coded in pkg/webhook/server/server.go
-          name: devworkspace-webhookserver
+          namespace: system
+          name: devworkspace-controller-manager-service
```

Comment:
Nope :) 

If you use the OLM functionality, it will assume that the webhooks service is called `<deployment-name>-service`, and create it if it does not exist. I originally had the suggested name, and installing via OLM would result in a duplicate service (with the exact same configuration) being created.

I think it's a bug in OLM. As far as I can tell, what happens is

1. Operator SDK detects that your CRDs have a the fields for `.spec.conversion.webhook.clientConfig.service` pointing to a service that matches a port on the deployment (all of this is required or SDK won't generate webhookDefinitions)
2. OperatorSDK generates a CSV that includes webhookDefinitions, which *don't* reference a service and instead reference the deployment in the manifests directly (`.webhookDefinitions[].deploymentName`)
3. OLM takes the bundle and creates `deploymentName-service` with matching port to support the webhook without looking to see if there's a different service.